extensions/BMO/template/en/default/hook/flag/type_comment-form.html.tmpl

```diff
@@ -0,0 +1,185 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+<template class="approval-request" data-flags="approval‑mozilla‑beta approval‑mozilla‑release">
```

Comment:
Are these the right flags?
What about approval-mozilla-central?

@emceeaich can you ask the stakeholder?

Reply:
I am the stakeholder for this. There is no such thing as approval-mozilla-central
this is beta, & release & esr.

I have some comments about the text/values. I will get back tomorrow.